easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.4.0-foss-2017b-Python-3.6.3.eb

```diff
@@ -0,0 +1,31 @@
+name = 'TensorFlow'
+version = '1.4.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.tensorflow.org/'
+description = "An open-source software library for Machine Intelligence"
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = ['https://github.com/tensorflow/tensorflow/archive/']
+sources = ['v%(version)s.tar.gz']
+patches = [
+    'TensorFlow-%(version)s_swig-env.patch',
+    'TensorFlow-%(version)s_no-enum34.patch',
+]
+checksums = [
+    '8a0ad8d61f8f6c0282c548661994a5ab83ac531bac496c3041dedc1ab021107b',  # v1.4.0.tar.gz
+    '53807290f1acb6a0f2a84f1a0ad9f917ee131c304b3e08679f3cbd335b22c7ef',  # TensorFlow-1.4.0_swig-env.patch
+    'aead2a8c2adee8bd39392edd27cdcbd114499219c8161dc6c6c5260b4b996495',  # TensorFlow-1.4.0_no-enum34.patch
+]
+
+builddependencies = [
+    ('Bazel', '0.7.0'),
+    ('wheel', '0.30.0', versionsuffix),
+]
+dependencies = [('Python', '3.6.3')]
+
+# FIXME
+runtest = False
```

Comment:
tests currently fail with:

```
== 2017-11-09 15:07:09,838 run.py:478 DEBUG cmd "bazel --output_base=%s test --config=opt //tensorflow/..." exited with exitcode 1 and output:
...........
____Loading package: tensorflow/java
____Loading package: tensorflow/contrib/input_pipeline
ERROR: infinite symlink expansion detected
[start of symlink chain]
/tmp/vsc40023/easybuild_build/TensorFlow/1.4.0/foss-2017b-Python-3.6.3/tensorflow-1.4.0/%s/external/org_tensorflow
/tmp/vsc40023/easybuild_build/TensorFlow/1.4.0/foss-2017b-Python-3.6.3/tensorflow-1.4.0
[end of symlink chain]
.
ERROR: error loading package 'tensorflow/compiler/aot': Encountered error while reading extension file 'tensorflow/tensorflow.bzl': no such package '@org_tensorflow//tensorflow': Could not access /tmp/vsc40023/easybuild_build/TensorFlow/1.4.0/foss-2017b-Python-3.6.3/tensorflow-1.4.0/%s/external/org_tensorflow: Infinite symlink expansion.
____Elapsed time: 1.414s
ERROR: Couldn't start the build. Unable to run tests.
```

Reply:
The symlink issue occurred because of a silly bug in the easyblock where `%s` was not completed with `tmpdir`, fixed now in https://github.com/easybuilders/easybuild-easyblocks/pull/1287.